strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/UserController.java

```diff
@@ -248,6 +241,27 @@ public ResponseEntity generateSecurityToken(@ApiParam(value = "The name of the u
 
         return ResponseEntity.ok(result);
     }
+
+    @ApiOperation(value = "Generate authentication token.", position = 3)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "The aouthentication token was generated."),
```

Comment:
Typo:
"The aouthentication token was generated."
Should be:
"The authentication token was generated."